package.json

```diff
@@ -132,7 +132,7 @@
       "git add"
     ],
     "**/*.rb": [
-      "bundle exec rubocop -a",
+      "bundle exec standard -a",
```

Comment:
Shouldn't it be `bundle exec standardrb` (or `bundle exec rake standard`)?

Reply:
Good catch, thanks @lidimayra!! 🙏